README.rdoc

```diff
@@ -206,15 +207,35 @@ you can do so with a simple environment variable, instead of editing the
             --frozen                     Do not allow to change annotations. Exits non-zero if there are going to be changes to files.
             --timestamp                  Include timestamp in (routes) annotation
             --trace                      If unable to annotate a file, print the full stack trace, not just the exception message.
-        -I, --ignore-columns REGEX       don't annotate columns that match a given REGEX (i.e., `annotate -I '^(id|updated_at|created_at)'`
-            --ignore-routes REGEX        don't annotate routes that match a given REGEX (i.e., `annotate -I '(mobile|resque|pghero)'`
+        -I, --ignore-columns REGEX       don't annotate columns that match a given REGEX (e.g. `annotate -I '^(id|updated_at|created_at)'`)
+            --ignore-routes REGEX        don't annotate routes that match a given REGEX (e.g. `annotate -I '(mobile|resque|pghero)'`)_
             --hide-limit-column-types VALUES
-                                         don't show limit for given column types, separated by commas (i.e., `integer,boolean,text`)
+                                         don't show limit for given column types, separated by commas (e.g. `integer,boolean,text`)
             --hide-default-column-types VALUES
-                                         don't show default for given column types, separated by commas (i.e., `json,jsonb,hstore`)
+                                         don't show default for given column types, separated by commas (e.g. `json,jsonb,hstore`)
             --ignore-unknown-models      don't display warnings for bad model files
             --with-comment               include database comments in model annotations
 
+=== Option: +additional_file_patterns+
+
+CLI: +--additional_file_patterns+<br>
+Ruby: +:additional_file_patterns+
+
+Provide additional paths for the gem to annotate.  These paths can include globs.
+It is recommended to use absolute paths.  Here are some examples:
+
+
+- <code>/app/lib/decorates/%MODEL_NAME%/*.rb</code>
+- <code>/app/lib/forms/%PLURALIZED_MODEL_NAME%/**/*.rb</code>
```

Comment:
@drwl I cannot figure out how to get `**` to escape correctly in rdoc rendering.  Any clue?  I've tried so many ways.

Reply:
Try `&ast;`, I believe it should work.

- <code>/app/lib/decorates/%MODEL_NAME%/*.rb</code>
- <code>/app/lib/forms/%PLURALIZED_MODEL_NAME%/&ast;&ast;/*.rb</code>

Source: https://github.com/adam-p/markdown-here/issues/287#issuecomment-282429074

Reply:
I actually can't replicate that.  Haha, I can't see the markdown for your comment so I can't copy it either.  Tried just using `&ast;` but that doesn't work either.  Mind just adding a commit?